frontend/src/components/Calendar.tsx

```diff
@@ -220,7 +225,7 @@ export default function Calendar({ yearData, setCurrentYear, currentYear }: Cale
                   <span
                     className=" h-2 w-2 rounded-full"
                     style={{
-                      backgroundColor: event.colorId ? colors[event.colorId] : "transparent",
+                      backgroundColor: event.colorId ? colors[event.colorId] : "#039be5",
```

Comment:
Make this consistent with the color of dot in calendar

Reply:
done.